### DISCUSSION TITLE
Feature for Graph Databases

### DISCUSSION THREAD

**Discussion by juniortads** [tryAGI/LangChain#409] [Ideas]
Hi everyone, any discussion about implementing support for graph-oriented databases? (example: Amazon Neptune, Neo4j) the same idea we have in LangChain (https://python.langchain.com/v0.1/docs/use_cases/graph/)

**Comment by HavenDV** [comment]
Unfortunately, we haven't even looked in this direction yet and will be for any help. But I think it will require correction/adding new abstractions to ensure this. You can always ask questions in discord and get a quick answer
